### DISCUSSION TITLE
Using svelvet library, can we have the edges connect to the input node from different positions?

### DISCUSSION THREAD

**Discussion by r-wadi** [open-source-labs/Svelvet#156] [Q&A] [answered]
[an example flow where the input node edges connect to it not in the same position ](https://i.stack.imgur.com/Q7Zwp.png)

Also if it's possible to have the label not on the edge line, but rather inside the node.

There doesn't seem to be a clear way to set the position of the each edge, nor to decide on the position of the label.

Side note, there doesn't seem to be a created #svelvet tag on stack overflow.

**Comment by thuyydt** [comment]
With current version, I just thinking about group the nodes. Put your tags to nodes and group them

**Comment by briangregoryholmes** [comment, marked as answer]
This is possible in Svelvet 7! https://svelvet.mintlify.app/components/anchor

Check out our new Sandbox for specific implementation details: https://svelvet.mintlify.app/introduction
